build.gradle

```diff
@@ -73,6 +73,7 @@ ext {
         commonsLang3:           '3.12.0',
         commonsText:            '1.9',
         commonsCollections4:    '4.4',
+        equalsverifier:         '3.5.5',
```

Comment:
it's better to keep the version copy-pasted in different modules, dependabot can update all of them within one PR (see, AWS modules as examples)